x-pack/filebeat/cmd/otel.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.opentelemetry.io/collector/otelcol"
+
+	"github.com/elastic/elastic-agent-libs/logp"
+)
+
+func OtelCmd() *cobra.Command {
```

Comment:
I think we want this command to be shared.

Reply:
This part of the code is dependent on `filebeatreceiver`  which is under x-pack license - so could not be moved to `beat/libbeat` folder.

Plus this logic is for the developer's testing purpose only. So I assumed it should be good for now

Reply:
we have `x-pack/libbeat`